packages/kmod-6.1-nvidia/nvidia-persistenced.service

```
@@ -0,0 +1,14 @@
+[Unit]
+Description=NVIDIA Persistence Daemon
+Requires=load-kernel-modules.service
+After=load-kernel-modules.service
+
+[Service]
+Type=forking
+# Run the NVIDIA System Management Interface to create the device nodes.
+ExecStart=/usr/libexec/nvidia/tesla/bin/nvidia-persistenced
+ExecStopPost=/bin/rm -rf /var/run/nvidia-persistenced
```

Would be good to clean up before as well, in case the service doesn't shut down cleanly:
```suggestion
ExecStartPre=/bin/rm -rf /run/nvidia-persistenced
ExecStart=/usr/libexec/nvidia/tesla/bin/nvidia-persistenced
ExecStopPost=/bin/rm -rf /run/nvidia-persistenced
```